sensor/kubernetes/listener/resources/dispatcher.go

```diff
@@ -202,6 +205,40 @@ type metricDispatcher struct {
 }
 
 func (m metricDispatcher) ProcessEvent(obj, oldObj interface{}, action central.ResourceAction) *component.ResourceEvent {
+	comp := env.InformerTraceLogs.Setting()
+	if action == central.ResourceAction_SYNC_RESOURCE || action == central.ResourceAction_REMOVE_RESOURCE {
+		switch v := obj.(type) {
+		case *v1rbac.Role:
+			if comp == "all" || comp == "role" {
+				log.Infof("Processing action %s for Role: %s", action.String(), v.UID)
+			}
+		case *v1rbac.RoleBinding:
+			if comp == "all" || comp == "rolebinding" {
+				log.Infof("Processing action %s for RoleBinding: %s", action.String(), v.UID)
+			}
+		case *v1rbac.ClusterRole:
+			if comp == "all" || comp == "clusterrole" {
+				log.Infof("Processing action %s for ClusterRole: %s", action.String(), v.UID)
+			}
+		case *v1rbac.ClusterRoleBinding:
+			if comp == "all" || comp == "clusterrolebinding" {
+				log.Infof("Processing action %s for ClusterRolebinding: %s", action.String(), v.UID)
+			}
+		case *v1.ServiceAccount:
+			if comp == "all" || comp == "serviceaccount" {
+				log.Infof("Processing action %s for ServiceAccount: %s", action.String(), v.UID)
+			}
+		case *v1.Secret:
+			if comp == "all" || comp == "secret" {
```

Comment:
Let's not forget to set the env var to just `secret` in the doc with the copy-paste commands.

Reply:
Thanks! Yes, I got it in by default.